packages/react/src/components/Header/_header.scss

```diff
@@ -126,3 +126,16 @@ button.#{$prefix}--header__menu-item {
     outline: none;
   }
 }
+
+.#{$iot-prefix}--header--story-test-class {
```

Comment:
Not sure about shipping styles that are just for the storybook to consumers. We can put these in the storybook css file.

Reply:
Moved to story scss file